datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/config/AppConfigResolver.java

```diff
@@ -90,14 +91,19 @@ public CompletableFuture<AppConfig> get(final DataFetchingEnvironment environmen
 
     final ManagedIngestionConfig ingestionConfig = new ManagedIngestionConfig();
     ingestionConfig.setEnabled(_ingestionConfiguration.isEnabled());
-    appConfig.setAuthConfig(authConfig);
```

Comment:
why is this being removed?

Reply:
It's there twice for some reason so I removed it when I got a merge conflict.